src/domain/common/whiteboard-rt/whiteboard-rt-authorization.service.ts

```diff
@@ -0,0 +1,127 @@
+import { Injectable } from '@nestjs/common';
+import {
+  AuthorizationCredential,
+  AuthorizationPrivilege,
+  LogContext,
+} from '@common/enums';
+import { IAuthorizationPolicy } from '@domain/common/authorization-policy';
+import { AuthorizationPolicyService } from '@domain/common/authorization-policy/authorization.policy.service';
+import { AuthorizationPolicyRulePrivilege } from '@core/authorization/authorization.policy.rule.privilege';
+import { EntityNotInitializedException } from '@common/exceptions/entity.not.initialized.exception';
+import { IAuthorizationPolicyRuleCredential } from '@core/authorization/authorization.policy.rule.credential.interface';
+import {
+  CREDENTIAL_RULE_WHITEBOARD_CREATED_BY,
+  CREDENTIAL_RULE_WHITEBOARD_RT_ACCESS,
+  POLICY_RULE_WHITEBOARD_CONTRIBUTE,
+  POLICY_RULE_WHITEBOARD_UPDATE,
+} from '@common/constants';
+import { ProfileAuthorizationService } from '../profile/profile.service.authorization';
+import { WhiteboardRtService } from './whiteboard.rt.service';
+import { IWhiteboardRt } from './whiteboard.rt.interface';
+
+@Injectable()
+export class WhiteboardRtAuthorizationService {
```

Comment:
filename has a different naming convention that in other modules, please follow the same one i.e. whiteboardrt.service.authorization